packages/utils/src/color/color.ts

```diff
@@ -0,0 +1,90 @@
+import { colord, extend } from 'colord';
+import cmykPlugin from 'colord/plugins/cmyk';
+import namesPlugin from 'colord/plugins/names';
+
+import type { AnyColor } from 'colord';
+import type { Color } from '@pixi/utils';
+
+extend([cmykPlugin, namesPlugin]);
```

Comment:
I don't think CMYK is necessary for us. WDYT?

Reply:
Perhaps, but the the cost of supporting cmyk is very small.